int16 tool_id 
---
geometry_msgs/PoseWithCovariance target_pose
string tool_name
bool success